store/boltdb/boltdb_test.go

```diff
@@ -0,0 +1,54 @@
+package boltdb
+
+import (
+	"testing"
+
+	"github.com/docker/libkv/store"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBoltDBStore(t *testing.T) {
```

Comment:
Agree.. better to use the existing tests. I created separately for the BoltDB because of the missing APIs. Instead I can group the tests into RunTestStore & RunTestStoreExtended and add the Watch, WatchTree etc. into the latter.